src/sage/rings/finite_rings/conway_polynomials.py

```diff
@@ -155,10 +162,13 @@ def __init__(self, p, use_database=True):
         TESTS::
 
             sage: from sage.rings.finite_rings.conway_polynomials import PseudoConwayLattice
+
+            sage: # needs sage.rings.finite_rings
             sage: PCL = PseudoConwayLattice(3)
```

Comment:
Why not 
```
sage: # needs sage.rings.finite_rings
sage: from sage.rings.finite_rings.conway_polynomials import PseudoConwayLattice
```
?

Reply:
I guess I wrote it this way because the following doctest also uses this import.
But I'll change it.

Reply:
Thanks.